Firestore/Source/API/FIRLocalCacheSettings.mm

```diff
@@ -73,7 +73,8 @@ - (instancetype)init {
 
 - (instancetype)initWithSizeBytes:(NSNumber *)size {
   self = [super init];
-  if (size.longLongValue < Settings::MinimumCacheSizeBytes) {
+  if (size.longLongValue != kFIRFirestoreCacheSizeUnlimited &&
```

Comment:
I thought we were going to use int64_t::max or something like that?

Also, please add a test to set kFIRFirestoreCacheSizeUnlimited

Reply:
As discussion today, using `-1` is more inline with other SDKs.

Test added.